modules/server/src/main/scala/navigate/server/acm/Decoder.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.server.acm
+
+trait Decoder[B, A] {
```

Comment:
We have so many `Decoder` classes, Would it be clearer e.g `AcmDecoder`?

Reply:
I prefer to use the simple names, and use explicit package names or import aliases to resolve conflicts.